staging/src/github.com/kubeedge/beehive/pkg/core/model/message.go

```diff
@@ -28,6 +29,7 @@ const (
 	ResourceTypePodlist             = "podlist"
 	ResourceTypePodStatus           = "podstatus"
 	ResourceTypeNodeStatus          = "nodestatus"
+	ResourceTypeNodePatch           = "nodepatch"
```

Comment:
this seems not a resource type, the resource is nodestatus and the operation is patch